src/local_options

```diff
@@ -24,7 +24,7 @@
 #undef NO_SNOOP
 #undef NO_ENVIRONMENT
 #undef NO_WIZARDS
-#undef NO_LIGHT
+#define NO_LIGHT
```

Comment:
Shouldn’t change this file, can you revert this?

Reply:
Copied the original back and recommitted under #[764ca52](https://github.com/fluffos/fluffos/pull/926/commits/764ca52ce7aa084d9058451a0dbc91293a9a1b14)